src/main/java/pico/typecheck/messages.properties

```diff
@@ -1,8 +1,8 @@
 constructor.invocation.invalid=Cannot not instantiate type: %s out of constructor: %s
 constructor.return.invalid=Invalid constructor return type: %s
-method.receiver.incompatible
-class.bound.invalid
-subclass.bound.incompatible
+method.receiver.incompatible=Incompatible method receiver: %s
+class.bound.invalid=Invalid class bound: %s
+subclass.bound.incompatible=Incompatible subclass bound: %s
```

Comment:
When is this error raised? Should the message be similar to `super.constructor.invocation.incompatible`, something like `Subclass bound %s is not compatible with super class bound %s`?
In general I find the term "subclass" strange and we should look to clean this terminology up. 
But let's do both of these improvements in separate PRs.